Dockerfile

```diff
@@ -16,6 +16,8 @@ RUN pip install pip --upgrade
 # Install project packages
 RUN pip install -r requirements.txt
 RUN pip install -r test_requirements.txt
+RUN pip3 install -r requirements.txt
+RUN pip3 install -r test_requirements.txt
```

Comment:
This was so I can runserver with python 3 for manual testing. Tox does everything on its own without it for 3.x